src/SwedbankPay.Sdk.Infrastructure/ProblemResponse.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace SwedbankPay.Sdk
+{
+    internal class ProblemResponse : IProblemResponse
+    {
+        public ProblemResponse(string action, string detail, string instance, List<IProblemResponseItem> problems, int status, string title, string type)

```

Comment:
Why are you not a fan of passing `List`?

Reply:
Because usually, the list can be just empty and thus omitted completely from construction. As long as the constructor then initializes it, it can be added items to it as needed. `Problems` is exactly such a list that can be empty and is not required to contain anything. Only when values are required should they be present in the constructor.